src/main/java/Brad/Brad.java

```diff
@@ -0,0 +1,253 @@
+                    } catch (ArrayIndexOutOfBoundsException | emptyArgumentException e) {
+                            printOutput("Uh oh. Please enter a number to mark the corresponding " +
+                                    "task as undone");
+                        } catch (invalidNumberException e) {
+                            printOutput("No! >:( Exceeded existing list size of: " + inputList.listSize() +
+                                            "\nPlease enter a valid number.\n");
+                    }
+                    break;
+                case "todo":
+                    try {
+                        doTodoAction(splitInput[1]);
+                    } catch (ArrayIndexOutOfBoundsException | emptyArgumentException e) {
+                        printOutput("Hey, you can't give me an empty todo!");
+                    }
+                    break;
+                case "deadline":
+                    try {
+                        doDeadlineAction(splitInput[1]);
+                    } catch (ArrayIndexOutOfBoundsException | emptyArgumentException e) {
+                        printOutput("Hey, you can't give me an empty deadline with no " +
+                                "specified due date!");
+                    }
+                    break;
+                case "event":
+                    try {
+                        doEventAction(splitInput[1]);
+                    } catch (ArrayIndexOutOfBoundsException | emptyArgumentException e) {
+                        printOutput("Hey, you can't give me an event with no start & end time!");
+                    }
+                    break;
+                case "delete":
+                    try {
+                        doDeleteAction(splitInput[1]);
+                    } catch (ArrayIndexOutOfBoundsException | emptyArgumentException e) {
+                        printOutput("Uh oh. Please enter a number to delete this task");
+                    } catch (invalidNumberException e) {
+                        printOutput("No! >:( Exceeded existing list size of: " + inputList.listSize()  +
+                                "\nPlease enter a valid number.\n");
+                    }
+                    break;
+                default:
+                    printOutput("Huh?! Sorry I don't understand. T_T\n Please only enter valid commands: " +
+                            "'list', 'mark', 'unmark', 'todo', 'deadline', 'event', 'delete', 'bye'");
+                    break;
+            }
+        }
+    }
+
+    private static void greetUser() {
+        final String name = "Brad";
+        System.out.println("Hello I am " + name + ".\n");
+        System.out.println("How can I help you today?\n");
+    }
+
+    private static void printOutput(String message) {
+        System.out.println(SEPARATOR);
+        System.out.println(message);
+        System.out.println(SEPARATOR);
+    }
+
+    private static void doMarkAction(String input) throws
```

Comment:
Can consider more OOP and adding a command abstract class with subclasses such as markcommand etc.